.gitignore

```diff
@@ -14,6 +14,8 @@ node_modules/
 .env.test.local
 .env.production.local
 
+config/
```

Comment:
I don't know if you'd want this whole folder to be gitignore? Maybe you use environment variables in config folder, and then ignore .env

Reply:
My mistake, I was under the impression that config was basically a way to do .env in .json. But I can see how there would be plenty of non-secret pieces of config data that would be worth comitting.